bin/collator/src/cli.rs

```diff
@@ -86,6 +86,14 @@ pub struct Cli {
     /// Relaychain arguments
     #[clap(raw = true)]
     pub relaychain_args: Vec<String>,
+
+    /// Proposer's maximum block size limit in bytes
+    #[clap(long)]
+    pub proposer_block_size_limit: Option<usize>,
```

Comment:
What’s about to have default values (based on relay expectations)?

Reply:
Unfortunately `DEFAULT_SOFT_DEADLINE_PERCENT` is not public. Of course, I can set the value by hand, but it would not be great IMO.

Reply:
I mean, sure, we can do something like this:

```rust
    /// Proposer's maximum block size limit in bytes
    #[clap(long, default_value = sc_basic_authorship::DEFAULT_BLOCK_SIZE_LIMIT)]
    pub proposer_block_size_limit: usize,

    /// Proposer's soft deadline in percents of block size
    #[clap(long, default_value = 50)]
    pub proposer_soft_deadline_percent: u8,
```

But that can be problematic if Parity would change the constant under the hood but we would still pass the original value by default.

Reply:
Looks good up to me, let's use it